v4/api/AbstractTest.java

```diff
@@ -1,98 +0,0 @@
-package com.equinix.openapi.fabric.v4.api;
```

Comment:
Is the folder v4/api all completely manually added by you? No code generation?

I don't know how this will be able to keep up because it's pulling a lot of code generated files over to it that are subject to change causing lots of overhead when updating tests.

Reply:
this class had been removed. these files must be located in separated folder because that is the java code structure. tests must be placed in defined place to run in during different maven command like test, package, publish etc. api code library generates only frames for each group of endpoints and they are without implementation